Vungle/VungleRouter.m

```diff
@@ -303,19 +321,24 @@ - (void)requestBannerAdWithPlacementID:(NSString *)placementID size:(CGSize)size
     }
 }
 
-- (BOOL)isAdAvailableForPlacementId:(NSString *) placementId {
+- (BOOL)isAdAvailableForPlacementId:(NSString *) placementId
```

Comment:
Extra space between `)` and `placementID`

Reply:
@crbinsf Thanks a lot for pointing out this error, fixed. :)